docker-compose.yml

```diff
@@ -133,7 +133,7 @@ services:
   postgres:
     <<: *restart_policy
     # Using the same postgres version as Sentry dev for consistency purposes
-    image: "postgres:14.11"
+    image: "postgres:14.19-bookworm"
```

Comment:
Any reason to specifically use `bookworm` here? Looks like 14.11 was on debian bookworm so not a big deal but I am scarred since siwtching to postgres alpine introduced a lot of errors for us

Reply:
14.11 was indeed debian bookworm. The default for `14.19` is debian trixie (debian 13). I don't know if that would cause any issues, but I would say that it's best to minimize issues for now.

And yes, we won't be moving to Alpine images.